Taskwarrior Pomodoro/AppDelegate.swift

```diff
@@ -212,9 +233,61 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
             pendingArguments = [definedDefaultFilter] + pendingArguments
         }
         
+        return getTasksUsingFilter(pendingArguments)
+    }
+    
+    func getTodaysPomodorosLog() -> JSON? {
+        let logFilter = ["status:Completed", kPomodoroLogEntryDescription, "entry:today", "limit:1"]
+        let tasks = getTasksUsingFilter(logFilter)
+        
+        guard !tasks.isEmpty else {
+            return nil
+        }
+        
+        currentPomodorosLogUUID = tasks[0]["uuid"].string
+        
+        return tasks[0]
+    }
+    
+    func getTasksUsingFilter(filter: [String]) -> [JSON] {
+        let arguments = ["rc.json.array=off"] + filter + ["export"]
+        
+        let output = taskCommandWithResult(arguments)
+        
+        let taskListStrings = output.characters.split{$0 == "\n"}.map(String.init)
+        
+        var taskList = [JSON]()
+        for taskListString in taskListStrings {
+            if let dataFromString = taskListString.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: true) {
+                let taskData = JSON(data: dataFromString)
+                taskList.append(taskData)
+            }
+        }
+        
+        return taskList;
+    }
+    
+    func createTodaysPomodorosLogEntry() -> String {
+        let arguments = ["log", kPomodoroLogEntryDescription]
+        let resultString = taskCommandWithResult(arguments)
```

Comment:
Ahh!  Yeah; I think we might be either running on different versions of Taskwarrior, or maybe have meaningfully different configurations.